website/src/categories/components/menu/usage.md

```diff
@@ -0,0 +1,148 @@
+</div>
+
+</section>
+
+<section>
+
+## When to Use
+The following guidance describes when to use the Menu component.
+
+### Contextual Actions
+Some interface elements can have additional actions, and providing a menu can enhance efficiency and streamline workflow. For instance, options to view details, open related documents, or access settings, offering extra details and actions specific to the element.
+
+### Reducing Visual Clutter
+Keeping a clean interface with repetitive elements (like cards or table rows), but additional actions are still necessary, a menu fixes this scenario while providing access to secondary actions. For example, sorting a list of elements or changing the view of the page.
+
+</section>
+
+
+<section>
+
+## When not to Use
+The following guidance describes when not to use the Menu component.
+
+### Primary Navigation
+Menus should not be used for primary navigation in applications because they are hidden until triggered by a specific action, making them less discoverable for users. Basic actions should always be visible and accessible to ensure users can efficiently navigate to essential features without additional steps. Relying on menus for primary navigation can also create inconsistency in accessing core functions by requiring users to learn multiple ways to perform basic tasks.
+
+</section>
+
+
+<section>
+
+## Anatomy
+
+|Item|Name| Description | Optional|
+|-|-|-|-|
+|1|Button Menu|This is the trigger button to open the menu. |no|
+|2|Menu Popover |This is the container for all the options of the menu, and appears close to the button menu. |no|
+|3|Option Items | This are the menu options that can be separete on "Option Button" that behave like a regular button, "Selection Option" that can behave like a checkbox or radio button, and "Sub-Menu Button" to open a nested menu. |no|
+|4|Group Heading |Title for the option groups to make eaiser to understand the function of the options. |yes|
+|5|Option Icons |You can display an icon in each option to enhance the readability. |yes|
+|6|Option Shortcuts |Adding shortcuts to the options to improve usability. |yes|
+|7|Group Heading |Title for the option groups to make eaiser to understand the function of the options. |yes|
+|8|Danger Option |Specific danger option of the menu. |yes|
+
+{.ds-table .ds-table-align-top}
+
+</section>
+
+
+<section>
+
+## Variants
+Menu comes in two versions, each suited for specific situations:
+
+  - **Button Option:** It triggers an immediate action or opens a submenu with further options. This variant is typically used for actions that have a direct and immediate effect, such as "Save," "Delete," or "Submit".
+
+  - **Single Select:** Allows users to choose only one option at a time. Clicking on an option deselects any previously selected items like a radio button. This variant is ideal for situations where users need to make mutually exclusive choices, such as change the sorting of a list or select the text editor in a text editor.
```

Comment:
Maybe it's good to describe the difference between this and a `select` component? Should a `select` not be used for navigation but should this one be used instead?

Reply:
@arecuenco-dsgn can you please take a look at this comment? :)

Reply:
This text describes the variant, and this scenario is covered in the "When not to use" section. The menu can't be used for navigation. It'll only be used to take actions related to the related element. 

This point can be explained better on the navigation pattern's page to cover all these scenarios.